### ISSUE TITLE
Enhancement: improve error handling for instant compilation

### ISSUE THREAD

**Issue by richardeschloss** [richardeschloss/nuxt-stories#69]
Instant compilation causes errors to occur as expected. Consider catching and squashing the known errors that occur.

**Comment by richardeschloss** [comment]
I started work on this and have a stub that helps using `Vue.config.errorHandler` but I think a better solution can be written once this PR https://github.com/vuejs/vue/pull/9482 is merged into Vue.

**Comment by richardeschloss** [comment]
Addressed in #78
